spec/features/invite_spec.rb

```diff
@@ -1,6 +1,6 @@
 require 'spec_helper'
 
-feature 'invitation' do
+feature 'invitation', {:vcr => true, :js => true} do
```

Comment:
Yes, new versions of RSpec allow you to pass just symbols instead of `symbol: true`.